app/org/sagebionetworks/bridge/dynamodb/DynamoParticipantOptionsDao.java

```diff
@@ -118,8 +116,6 @@ public OptionLookup getOptionForAllStudyParticipants(StudyIdentifier studyIdenti
         scan.addFilterCondition("studyKey", condition);
         
         List<DynamoParticipantOptions> mappings = mapper.scan(DynamoParticipantOptions.class, scan);
-        List<FailedBatch> failures = mapper.batchDelete(mappings);
-        BridgeUtils.ifFailuresThrowException(failures);
```

Comment:
We should do something because this was a scary mistake had we used it on production. A mock that verifies no other interactions would be good. Should do it elsewhere too.